client/src/pages/Dashboard.js

```diff
@@ -40,46 +38,60 @@ function Dashboard() {
     axios
       .get(`/settings/${localStorage.getItem("email")}`)
       .then(res => {
-        if (res.data.success) {
+        if (res.data.authenticated === false) {
+          handleLogout();
```

Comment:
Can we have a wrapper for axios that performs that check for every authenticated request?